src/components/props-panel/index.tsx

```diff
@@ -26,7 +31,20 @@ export default function PropsPanel({canvas}: Props) {
   const onDeleteSelected = () => {
     canvas?.deleteSelected()
   }
-  
+
+  const fontArr: string[] = [
+    "Arial",
+    "Times New Roman",
```

Comment:
Are these fonts windows/mac support natively ?

Reply:
Yeah, I asked chatgpt for the common ones haha